steps/redhat.py

```diff
@@ -230,3 +236,19 @@ def step_impl(context, pattern):
 	pattern_occurence =  context.remote_cmd(cmd='command',
                     module_args='sudo find / \( -path "/proc" -o -path "/sys" -o -path "/dev" -o -path "/sysroot" -o -path "/var/home" \) -prune -o -type f -exec grep -nHI "' + pattern + '" {} \;')
 	assert pattern_occurence == "", "Fail, the " + pattern + " is present on the system."
+
+@when(u'IP address of the "{host}" is recorded')
+def step_impl(context, host):
+	ip_addr = get_ipaddr(context, host)
```

Comment:
It would probably be better to save this in the context instead. E.g. `context.recorded_ip = get_ipaddr(context, host)`.